tests/x11/firefox/firefox_passwd.pm

```diff
@@ -89,14 +90,21 @@ sub run {
     assert_screen('firefox-passwd-saved');
 
     send_key "alt-shift-a";    #"Remove"
-    wait_still_screen 3;
+    assert_and_click('firefox-saved-logins-remove') unless is_sle('=15');
```

Comment:
What about Leap?

Reply:
> What about Leap?

The thing is that I didn't manage to find these regression tests scheduled on ooo, so I took into consideration the modification of the tests that are scheduled right now.
Feel free though to indicate the location of the corresponding tests in ooo, in order for me to be able to do the cross-check and validate the behavior on TW and Leap too, as commented above also by Oliver.

Reply:
I don't think I've ever seen `firefox_passwd` running on o3, do you know why? Factory first is a thing...

Still, it doesn't make sense to me to make this SLE specific now, as it most definitely would break on Leap then.

Reply:
done ;)